train.py

```diff
@@ -489,6 +518,10 @@ def run(cfg: Any) -> None:
         cfg: config object with all the hyperparameters
     """
 
+    if cfg.environment.use_deepspeed:
+        # To avoid warnings about parallelism in tokenizers
+        os.environ["TOKENIZERS_PARALLELISM"] = "false"
+
```

Comment:
should already be set on the very top of this file. Why again?

Reply:
[d30b51c](https://github.com/h2oai/h2o-llmstudio/pull/288/commits/d30b51c36d7dc3de5814bc561db826c196ce615b)

fixed.